### DISCUSSION TITLE
Transfer Samples with Oauth

### DISCUSSION THREAD

**Discussion by johnnychoque** [eclipse-edc/Samples#210] [Q&A] [answered]
Hello all,

I noticed that Transfer Samples uses the iam-mock extension. I would like to run Transfer Samples with an external IAM, such as Keycloak, for which I understand I must use the oauth2 extension. Is there any documentation explaining how to configure and use that extension?

Regards,

Johnny

**Comment by ndr-brt** [comment, marked as answer]
`iam-mock` replaces the whole "identity management", that's a completely dataspace specific component, some of them uses DAPS (centralized), but some are moving to the Identity and Trust Protocol.
You can see a sample that is currently using DID WEB in the [MinimumViableDataspace](https://github.com/eclipse-edc/MinimumViableDataspace), or, a specific one on the CatenaX dataspace called [MXD](https://github.com/eclipse-tractusx/tutorial-resources/tree/main/mxd).

TL;DR: identity management is not in the scope of these samples
